docs/troubleshoot/verifier/notation.md

```diff
@@ -1,8 +1,32 @@
 # Troubleshoot Notation Verifier Errors
 
-If an image verification failed for some reason, please check the logs in Ratify pod to inspect the related error logs.
+There are generally two types of errors that can occur when using the Notation verifier: configuration errors and verification errors. Configuration errors occur when the Notation verifier is not configured correctly. Verification errors occur when the Notation verifier fails to verify an artifact. This TSG provides guidance on how to troubleshoot Notation verifier errors.
```

Comment:
Suggestion: This TSG provides guidance on how to troubleshoot both types of Notation verifier errors.